components/inputs/demo/input-number.html

```diff
@@ -25,6 +25,13 @@ <h2>Hidden Label</h2>
 					<d2l-input-number label="Age" label-hidden></d2l-input-number>
 				</template>
 			</d2l-demo-snippet>
+			
+			<h2>Hidden Label Different from Visible Label</h2>
+			<d2l-demo-snippet>
+				<template>
+					<d2l-input-number label="Age" hidden-label="Age (number)"></d2l-input-number>
```

Comment:
This isn't really a good example of a scenario you'd want to use this, so let me know if anyone has better ideas.

Reply:
I think the only valid reason to use this is when sighted users get more context from visual layout, and we want to provide that context to screen reader users (and even then, this can often be addressed with a labelled fieldset or heading). So I think we should use an example that _doesn't_ suggest using this to include help text or units, since there are better ways to do both. How about "Age (additional context needed for screen reader users)"?